stroeer/core/v1/core_article_service.proto

```diff
@@ -10,10 +10,42 @@ option java_package = "de.stroeer.core.v1";
 
 // Service to manage core articles
 service ArticleService {
-  // Returns a core article by it's id
+  // Returns a core article by it's `id`
   rpc GetArticle (GetArticleRequest) returns (stroeer.core.v1.Article) {}
+
+  // Returns pages of articles by their `home_section`
+  rpc ListArticles (ListArticlesRequest) returns (ListArticlesResponse) {}
+  // also as continuous stream
+  rpc LotsOfArticles (ListArticlesRequest) returns (stream ListArticlesResponse) {}
 }
 
 message GetArticleRequest {
   string id = 1;
 }
+
+message ListArticlesRequest {
+  // mandatory: valid section_path, with leading and trailing slash
+  // e.g. `/sport/`
+  string section_path = 1;
+
+  // The maximum number of books to return. The service may return fewer than
+  // this value.
+  // If unspecified, at most 10 articles will be returned.
+  // The maximum value is 50; values above 1000 will be coerced to 1000.
```

Comment:
c&p from AIP ;) We sell articles not books

Shall we use 100 as the max value for each page (~ what one ddb batch request supports) and 30 as default if unspecified?

Reply:
the ddb limits are `16 MB` or `100` items. so yeah, lets allow 100.

Reply:
Elasticsearch can handle upto `10k` hits. Exposing this that returns upto `1000` maybe ok. I like the min `10` and max `100` set for the start.

Reply:
we're not using ES behind the scenes, that's why we go for 100. the user has to progress through the result set, I think this is ok and works as expected.